cypress/e2e/GUI/customer_details.cy.js

```diff
@@ -0,0 +1,50 @@
+/// <reference types="Cypress" />
+describe('Detalhes do cliente', () => {
+    beforeEach(() => {
+        const dateInThePast = new Date(Date.UTC(1983, 6, 7))
+            cy.clock(dateInThePast)
+            cy.visit('/') 
+    });
+    context('Quando valido a saudação por default e a data', () => {
+        it('Então deve conter "Hi there" e a data "Wed Jul 06 1983"', () => {
+            cy.wait('@pag1limit10all')
+            cy.get('[data-testid="table"] tbody tr').then(($confirmElement) => {
+                if($confirmElement.length > 0) {
+                    cy.get('[data-testid="table"] > :nth-child(1)')
```

Comment:
Que tal algo mais simples, como `cy.contains('p', 'Hi there').should('be.visible')`?